core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -139,6 +198,144 @@ var StoryEditorPage = function() {
     closeSaveModalButton.click();
     waitFor.pageToFullyLoad();
   };
+
+  this.expectSaveStoryDisabled = function() {
+    return expect(
+      saveStoryButton.getAttribute('disabled')).toEqual('true');
+  };
+
+  this.expectDisplayUnreachableChapter = function() {
+    return expect(disconnectedChapterWarning.isPresent()).toBe(true);
+  };
+
+  this.setChapterExplorationId = function(explorationId) {
+    waitFor.visibilityOf(
+      explorationIdInput,
+      'ExplorationIdInput takes too long to be visible'
+    );
+
+    explorationIdInput.sendKeys(explorationId);
+    waitFor.elementToBeClickable(
+      explorationIdSaveButton,
+      'ExplorationIdSaveButton takes too long to be visible'
+    );
+    explorationIdSaveButton.click();
+  };
+
+  this.changeNodeOutline = function(richTextInstructions) {
+    var editor = forms.RichTextEditor(
+      nodeOutlineEditor);
+    editor.clear();
+    richTextInstructions(editor);
+    nodeOutlineSaveButton.click();
+  };
+
+  this.navigateToChapterByIndex = function(index) {
+    chapterTitles.then(function(elements) {
+      elements[index].click();
+    });
+  };
+
+  this.expectExplorationIdAlreadyExistWarning = function() {
+    var warningToast = element(by.css('.toast-message'));
+    waitFor.visibilityOf(
+      warningToast,
+      'warningToast takes too long to be visible.');
+    expect(warningToast.getText()).toEqual(
+      'The given exploration already exists in the story.');
+    var closeToastButton = element(by.css('.toast-close-button'));
+    waitFor.elementToBeClickable(
+      closeToastButton,
+      'closeToastButton takes too long to be clickable.');
+    closeToastButton.click();
```

Comment:
One wouldn't expect from the function name that it closes the toast. The name suggests that it just checks that the toast is showing the proper warning. I would either:
* Close the toast from outside this function
* Change the name to clarify that it also closes the toast

Reply:
Updated the name